diff -urN zipios++-0.1.5/src/zipios_common.h zipios++-0.1.5.new/src/zipios_common.h
--- zipios++-0.1.5/src/zipios_common.h	2000-08-04 15:26:00.000000000 +0200
+++ zipios++-0.1.5.new/src/zipios_common.h	2006-12-24 03:52:40.872318426 +0100
@@ -13,7 +13,7 @@
 
 template< class Type >
 void operator += ( vector< Type > &v1, const vector< Type > &v2 ) {
-  std::vector<Type>::const_iterator cit ;
+  class vector<Type>::const_iterator cit ;
   for ( cit = v2.begin() ; cit != v2.end() ; cit++ )
     v1.push_back( *cit ) ;
 }
